### DISCUSSION TITLE
Images don't stitch despite high enough confidence

### DISCUSSION THREAD

**Discussion by neriniccolodei** [OpenStitching/stitching#155] [General]
Hello,

I believe I read the same issue somewhere. 
I am stitching images for which the following match graph is printed:
`graph matches_graph{
"frame075.jpg" -- "frame076.jpg"[label="Nm=54, Ni=35, C=1.44628"];
"frame076.jpg" -- "frame077.jpg"[label="Nm=43, Ni=26, C=1.24402"];
"frame077.jpg" -- "frame078.jpg"[label="Nm=49, Ni=37, C=1.62996"];
"frame078.jpg" -- "frame079.jpg"[label="Nm=48, Ni=36, C=1.60714"];
"frame079.jpg" -- "frame080.jpg"[label="Nm=48, Ni=36, C=1.60714"];
"frame080.jpg" -- "frame081.jpg"[label="Nm=46, Ni=33, C=1.51376"];
"frame081.jpg" -- "frame082.jpg"[label="Nm=37, Ni=30, C=1.57068"];
"frame082.jpg" -- "frame083.jpg"[label="Nm=48, Ni=35, C=1.5625"]; 
"frame083.jpg" -- "frame084.jpg"[label="Nm=56, Ni=44, C=1.77419"];
"frame084.jpg" -- "frame085.jpg"[label="Nm=42, Ni=32, C=1.5534"]; 
"frame085.jpg" -- "frame086.jpg"[label="Nm=49, Ni=38, C=1.67401"];
"frame086.jpg" -- "frame087.jpg"[label="Nm=49, Ni=40, C=1.76211"];
"frame087.jpg" -- "frame088.jpg"[label="Nm=41, Ni=31, C=1.52709"];
"frame088.jpg" -- "frame089.jpg"[label="Nm=40, Ni=36, C=1.8"];    
"frame089.jpg" -- "frame090.jpg"[label="Nm=44, Ni=30, C=1.41509"];
"frame090.jpg" -- "frame091.jpg"[label="Nm=29, Ni=25, C=1.49701"];
"frame091.jpg" -- "frame092.jpg"[label="Nm=32, Ni=22, C=1.25"];   
"frame092.jpg" -- "frame093.jpg"[label="Nm=28, Ni=21, C=1.28049"];
"frame093.jpg" -- "frame094.jpg"[label="Nm=22, Ni=18, C=1.23288"];
"frame094.jpg" -- "frame095.jpg"[label="Nm=32, Ni=26, C=1.47727"];
"frame095.jpg" -- "frame096.jpg"[label="Nm=38, Ni=30, C=1.54639"];
"frame097.jpg" -- "frame098.jpg"[label="Nm=32, Ni=28, C=1.59091"];
"frame098.jpg" -- "frame099.jpg"[label="Nm=42, Ni=32, C=1.5534"]; 
"frame099.jpg" -- "frame100.jpg"[label="Nm=47, Ni=35, C=1.58371"];
"frame100.jpg" -- "frame101.jpg"[label="Nm=44, Ni=38, C=1.79245"];
"frame101.jpg" -- "frame102.jpg"[label="Nm=41, Ni=32, C=1.57635"];
"frame102.jpg" -- "frame103.jpg"[label="Nm=39, Ni=33, C=1.67513"];
"frame103.jpg" -- "frame104.jpg"[label="Nm=42, Ni=32, C=1.5534"];
"frame104.jpg" -- "frame105.jpg"[label="Nm=46, Ni=39, C=1.78899"];
"frame105.jpg" -- "frame106.jpg"[label="Nm=47, Ni=41, C=1.8552"];
"frame106.jpg" -- "frame107.jpg"[label="Nm=55, Ni=45, C=1.83673"];
"frame107.jpg" -- "frame108.jpg"[label="Nm=66, Ni=44, C=1.58273"];
"frame108.jpg" -- "frame109.jpg"[label="Nm=54, Ni=46, C=1.90083"];
"frame109.jpg" -- "frame110.jpg"[label="Nm=56, Ni=42, C=1.69355"];
"frame110.jpg" -- "frame111.jpg"[label="Nm=65, Ni=47, C=1.70909"];
"frame111.jpg" -- "frame112.jpg"[label="Nm=84, Ni=69, C=2.07831"];
"frame112.jpg" -- "frame113.jpg"[label="Nm=101, Ni=81, C=2.11488"];
"frame113.jpg" -- "frame114.jpg"[label="Nm=85, Ni=75, C=2.23881"];
"frame114.jpg" -- "frame115.jpg"[label="Nm=81, Ni=65, C=2.01238"];
"frame115.jpg" -- "frame116.jpg"[label="Nm=76, Ni=61, C=1.98052"];
"frame116.jpg" -- "frame117.jpg"[label="Nm=78, Ni=63, C=2.00637"];
"frame117.jpg" -- "frame118.jpg"[label="Nm=72, Ni=54, C=1.82432"];
"frame118.jpg" -- "frame119.jpg"[label="Nm=64, Ni=50, C=1.83824"];
"frame119.jpg" -- "frame120.jpg"[label="Nm=61, Ni=44, C=1.673"];
"frame120.jpg" -- "frame121.jpg"[label="Nm=72, Ni=55, C=1.85811"];
"frame121.jpg" -- "frame122.jpg"[label="Nm=72, Ni=41, C=1.38514"];
"frame122.jpg" -- "frame123.jpg"[label="Nm=72, Ni=54, C=1.82432"];
"frame123.jpg" -- "frame124.jpg"[label="Nm=74, Ni=51, C=1.68874"];
"frame124.jpg" -- "frame125.jpg"[label="Nm=75, Ni=46, C=1.5082"];
"frame125.jpg" -- "frame126.jpg"[label="Nm=91, Ni=77, C=2.1813"];
"frame126.jpg" -- "frame127.jpg"[label="Nm=89, Ni=75, C=2.16138"];
"frame127.jpg" -- "frame128.jpg"[label="Nm=89, Ni=70, C=2.01729"];
"frame128.jpg" -- "frame129.jpg"[label="Nm=73, Ni=68, C=2.27425"];
"frame129.jpg" -- "frame130.jpg"[label="Nm=71, Ni=59, C=2.01365"];
"frame130.jpg" -- "frame131.jpg"[label="Nm=68, Ni=54, C=1.90141"];
"frame131.jpg" -- "frame132.jpg"[label="Nm=70, Ni=54, C=1.86207"];
"frame132.jpg" -- "frame133.jpg"[label="Nm=77, Ni=64, C=2.05788"];
"frame133.jpg" -- "frame134.jpg"[label="Nm=89, Ni=69, C=1.98847"];
"frame134.jpg" -- "frame135.jpg"[label="Nm=92, Ni=63, C=1.76966"];
"frame135.jpg" -- "frame136.jpg"[label="Nm=82, Ni=55, C=1.68712"];
"frame136.jpg" -- "frame137.jpg"[label="Nm=80, Ni=64, C=2"];
"frame137.jpg" -- "frame138.jpg"[label="Nm=99, Ni=81, C=2.14854"];
"frame138.jpg" -- "frame139.jpg"[label="Nm=96, Ni=81, C=2.20109"];
"frame139.jpg" -- "frame140.jpg"[label="Nm=99, Ni=65, C=1.72414"];
"frame140.jpg" -- "frame141.jpg"[label="Nm=104, Ni=79, C=2.01531"];
"frame141.jpg" -- "frame142.jpg"[label="Nm=97, Ni=76, C=2.04852"];
"frame142.jpg" -- "frame143.jpg"[label="Nm=92, Ni=66, C=1.85393"];
"frame143.jpg" -- "frame144.jpg"[label="Nm=92, Ni=74, C=2.07865"];
"frame144.jpg" -- "frame145.jpg"[label="Nm=100, Ni=76, C=2"];
"frame145.jpg" -- "frame146.jpg"[label="Nm=99, Ni=67, C=1.77719"];
"frame146.jpg" -- "frame147.jpg"[label="Nm=109, Ni=89, C=2.18673"];
"frame147.jpg" -- "frame148.jpg"[label="Nm=132, Ni=96, C=2.01681"];
"frame148.jpg" -- "frame149.jpg"[label="Nm=135, Ni=93, C=1.91753"];
"frame149.jpg" -- "frame150.jpg"[label="Nm=131, Ni=88, C=1.86047"];
"frame150.jpg" -- "frame151.jpg"[label="Nm=152, Ni=101, C=1.88433"];
"frame151.jpg" -- "frame152.jpg"[label="Nm=141, Ni=100, C=1.98807"];
"frame152.jpg" -- "frame153.jpg"[label="Nm=141, Ni=96, C=1.90855"];
"frame153.jpg" -- "frame154.jpg"[label="Nm=118, Ni=86, C=1.98157"];
"frame154.jpg" -- "frame155.jpg"[label="Nm=103, Ni=82, C=2.10797"];
"frame155.jpg" -- "frame156.jpg"[label="Nm=100, Ni=59, C=1.55263"];
"frame156.jpg" -- "frame157.jpg"[label="Nm=110, Ni=78, C=1.90244"];
"frame157.jpg" -- "frame158.jpg"[label="Nm=111, Ni=73, C=1.76755"];
"frame158.jpg" -- "frame159.jpg"[label="Nm=149, Ni=88, C=1.66983"];
"frame159.jpg" -- "frame160.jpg"[label="Nm=144, Ni=109, C=2.12891"];
"frame160.jpg" -- "frame161.jpg"[label="Nm=151, Ni=96, C=1.80113"];
"frame161.jpg" -- "frame162.jpg"[label="Nm=145, Ni=87, C=1.68932"];
"frame162.jpg" -- "frame163.jpg"[label="Nm=131, Ni=71, C=1.50106"];
"frame163.jpg" -- "frame164.jpg"[label="Nm=123, Ni=80, C=1.78174"];
"frame164.jpg" -- "frame165.jpg"[label="Nm=97, Ni=65, C=1.75202"];
"frame165.jpg" -- "frame166.jpg"[label="Nm=115, Ni=65, C=1.52941"];
"frame166.jpg" -- "frame167.jpg"[label="Nm=112, Ni=64, C=1.53846"];
"frame167.jpg" -- "frame168.jpg"[label="Nm=110, Ni=73, C=1.78049"];
"frame168.jpg" -- "frame169.jpg"[label="Nm=102, Ni=61, C=1.58031"];
"frame169.jpg" -- "frame170.jpg"[label="Nm=95, Ni=50, C=1.36986"];
"frame170.jpg" -- "frame171.jpg"[label="Nm=124, Ni=69, C=1.52655"];
"frame171.jpg" -- "frame172.jpg"[label="Nm=112, Ni=68, C=1.63462"];
"frame172.jpg" -- "frame173.jpg"[label="Nm=88, Ni=59, C=1.71512"];
"frame173.jpg" -- "frame174.jpg"[label="Nm=86, Ni=56, C=1.6568"];
"frame174.jpg" -- "frame175.jpg"[label="Nm=84, Ni=60, C=1.80723"];
"frame175.jpg" -- "frame176.jpg"[label="Nm=90, Ni=74, C=2.11429"];
"frame176.jpg" -- "frame177.jpg"[label="Nm=98, Ni=60, C=1.60428"];
}`

With `confidence_threshold = 1`, I expect all frames to be stitched together since each C value is higher than the threshold. I am using `range_width=1`, orb, 1000 features. 
Still, the resulting image is a stitch from around `frame96.jpg` until the end. Therefore, no information from previous frames is used in the final image. 

Why would that be?

[Here](https://we.tl/t-iWO5xR0f2w)'s the dataset I am using.

**Comment by lukasalexanderweber** [comment]
A match between `frame096.jpg -- frame097.jpg` is missing. See the [matches_graph](https://dreampuf.github.io/GraphvizOnline/#%20graph%20matches_graph%7B%20%22frame075.jpg%22%20--%20%22frame076.jpg%22%5Blabel%3D%22Nm%3D54%2C%20Ni%3D35%2C%20C%3D1.44628%22%5D%3B%20%22frame076.jpg%22%20--%20%22frame077.jpg%22%5Blabel%3D%22Nm%3D43%2C%20Ni%3D26%2C%20C%3D1.24402%22%5D%3B%20%22frame077.jpg%22%20--%20%22frame078.jpg%22%5Blabel%3D%22Nm%3D49%2C%20Ni%3D37%2C%20C%3D1.62996%22%5D%3B%20%22frame078.jpg%22%20--%20%22frame079.jpg%22%5Blabel%3D%22Nm%3D48%2C%20Ni%3D36%2C%20C%3D1.60714%22%5D%3B%20%22frame079.jpg%22%20--%20%22frame080.jpg%22%5Blabel%3D%22Nm%3D48%2C%20Ni%3D36%2C%20C%3D1.60714%22%5D%3B%20%22frame080.jpg%22%20--%20%22frame081.jpg%22%5Blabel%3D%22Nm%3D46%2C%20Ni%3D33%2C%20C%3D1.51376%22%5D%3B%20%22frame081.jpg%22%20--%20%22frame082.jpg%22%5Blabel%3D%22Nm%3D37%2C%20Ni%3D30%2C%20C%3D1.57068%22%5D%3B%20%22frame082.jpg%22%20--%20%22frame083.jpg%22%5Blabel%3D%22Nm%3D48%2C%20Ni%3D35%2C%20C%3D1.5625%22%5D%3B%20%20%22frame083.jpg%22%20--%20%22frame084.jpg%22%5Blabel%3D%22Nm%3D56%2C%20Ni%3D44%2C%20C%3D1.77419%22%5D%3B%20%22frame084.jpg%22%20--%20%22frame085.jpg%22%5Blabel%3D%22Nm%3D42%2C%20Ni%3D32%2C%20C%3D1.5534%22%5D%3B%20%20%22frame085.jpg%22%20--%20%22frame086.jpg%22%5Blabel%3D%22Nm%3D49%2C%20Ni%3D38%2C%20C%3D1.67401%22%5D%3B%20%22frame086.jpg%22%20--%20%22frame087.jpg%22%5Blabel%3D%22Nm%3D49%2C%20Ni%3D40%2C%20C%3D1.76211%22%5D%3B%20%22frame087.jpg%22%20--%20%22frame088.jpg%22%5Blabel%3D%22Nm%3D41%2C%20Ni%3D31%2C%20C%3D1.52709%22%5D%3B%20%22frame088.jpg%22%20--%20%22frame089.jpg%22%5Blabel%3D%22Nm%3D40%2C%20Ni%3D36%2C%20C%3D1.8%22%5D%3B%20%20%20%20%20%22frame089.jpg%22%20--%20%22frame090.jpg%22%5Blabel%3D%22Nm%3D44%2C%20Ni%3D30%2C%20C%3D1.41509%22%5D%3B%20%22frame090.jpg%22%20--%20%22frame091.jpg%22%5Blabel%3D%22Nm%3D29%2C%20Ni%3D25%2C%20C%3D1.49701%22%5D%3B%20%22frame091.jpg%22%20--%20%22frame092.jpg%22%5Blabel%3D%22Nm%3D32%2C%20Ni%3D22%2C%20C%3D1.25%22%5D%3B%20%20%20%20%22frame092.jpg%22%20--%20%22frame093.jpg%22%5Blabel%3D%22Nm%3D28%2C%20Ni%3D21%2C%20C%3D1.28049%22%5D%3B%20%22frame093.jpg%22%20--%20%22frame094.jpg%22%5Blabel%3D%22Nm%3D22%2C%20Ni%3D18%2C%20C%3D1.23288%22%5D%3B%20%22frame094.jpg%22%20--%20%22frame095.jpg%22%5Blabel%3D%22Nm%3D32%2C%20Ni%3D26%2C%20C%3D1.47727%22%5D%3B%20%22frame095.jpg%22%20--%20%22frame096.jpg%22%5Blabel%3D%22Nm%3D38%2C%20Ni%3D30%2C%20C%3D1.54639%22%5D%3B%20%22frame097.jpg%22%20--%20%22frame098.jpg%22%5Blabel%3D%22Nm%3D32%2C%20Ni%3D28%2C%20C%3D1.59091%22%5D%3B%20%22frame098.jpg%22%20--%20%22frame099.jpg%22%5Blabel%3D%22Nm%3D42%2C%20Ni%3D32%2C%20C%3D1.5534%22%5D%3B%20%20%22frame099.jpg%22%20--%20%22frame100.jpg%22%5Blabel%3D%22Nm%3D47%2C%20Ni%3D35%2C%20C%3D1.58371%22%5D%3B%20%22frame100.jpg%22%20--%20%22frame101.jpg%22%5Blabel%3D%22Nm%3D44%2C%20Ni%3D38%2C%20C%3D1.79245%22%5D%3B%20%22frame101.jpg%22%20--%20%22frame102.jpg%22%5Blabel%3D%22Nm%3D41%2C%20Ni%3D32%2C%20C%3D1.57635%22%5D%3B%20%22frame102.jpg%22%20--%20%22frame103.jpg%22%5Blabel%3D%22Nm%3D39%2C%20Ni%3D33%2C%20C%3D1.67513%22%5D%3B%20%22frame103.jpg%22%20--%20%22frame104.jpg%22%5Blabel%3D%22Nm%3D42%2C%20Ni%3D32%2C%20C%3D1.5534%22%5D%3B%20%22frame104.jpg%22%20--%20%22frame105.jpg%22%5Blabel%3D%22Nm%3D46%2C%20Ni%3D39%2C%20C%3D1.78899%22%5D%3B%20%22frame105.jpg%22%20--%20%22frame106.jpg%22%5Blabel%3D%22Nm%3D47%2C%20Ni%3D41%2C%20C%3D1.8552%22%5D%3B%20%22frame106.jpg%22%20--%20%22frame107.jpg%22%5Blabel%3D%22Nm%3D55%2C%20Ni%3D45%2C%20C%3D1.83673%22%5D%3B%20%22frame107.jpg%22%20--%20%22frame108.jpg%22%5Blabel%3D%22Nm%3D66%2C%20Ni%3D44%2C%20C%3D1.58273%22%5D%3B%20%22frame108.jpg%22%20--%20%22frame109.jpg%22%5Blabel%3D%22Nm%3D54%2C%20Ni%3D46%2C%20C%3D1.90083%22%5D%3B%20%22frame109.jpg%22%20--%20%22frame110.jpg%22%5Blabel%3D%22Nm%3D56%2C%20Ni%3D42%2C%20C%3D1.69355%22%5D%3B%20%22frame110.jpg%22%20--%20%22frame111.jpg%22%5Blabel%3D%22Nm%3D65%2C%20Ni%3D47%2C%20C%3D1.70909%22%5D%3B%20%22frame111.jpg%22%20--%20%22frame112.jpg%22%5Blabel%3D%22Nm%3D84%2C%20Ni%3D69%2C%20C%3D2.07831%22%5D%3B%20%22frame112.jpg%22%20--%20%22frame113.jpg%22%5Blabel%3D%22Nm%3D101%2C%20Ni%3D81%2C%20C%3D2.11488%22%5D%3B%20%22frame113.jpg%22%20--%20%22frame114.jpg%22%5Blabel%3D%22Nm%3D85%2C%20Ni%3D75%2C%20C%3D2.23881%22%5D%3B%20%22frame114.jpg%22%20--%20%22frame115.jpg%22%5Blabel%3D%22Nm%3D81%2C%20Ni%3D65%2C%20C%3D2.01238%22%5D%3B%20%22frame115.jpg%22%20--%20%22frame116.jpg%22%5Blabel%3D%22Nm%3D76%2C%20Ni%3D61%2C%20C%3D1.98052%22%5D%3B%20%22frame116.jpg%22%20--%20%22frame117.jpg%22%5Blabel%3D%22Nm%3D78%2C%20Ni%3D63%2C%20C%3D2.00637%22%5D%3B%20%22frame117.jpg%22%20--%20%22frame118.jpg%22%5Blabel%3D%22Nm%3D72%2C%20Ni%3D54%2C%20C%3D1.82432%22%5D%3B%20%22frame118.jpg%22%20--%20%22frame119.jpg%22%5Blabel%3D%22Nm%3D64%2C%20Ni%3D50%2C%20C%3D1.83824%22%5D%3B%20%22frame119.jpg%22%20--%20%22frame120.jpg%22%5Blabel%3D%22Nm%3D61%2C%20Ni%3D44%2C%20C%3D1.673%22%5D%3B%20%22frame120.jpg%22%20--%20%22frame121.jpg%22%5Blabel%3D%22Nm%3D72%2C%20Ni%3D55%2C%20C%3D1.85811%22%5D%3B%20%22frame121.jpg%22%20--%20%22frame122.jpg%22%5Blabel%3D%22Nm%3D72%2C%20Ni%3D41%2C%20C%3D1.38514%22%5D%3B%20%22frame122.jpg%22%20--%20%22frame123.jpg%22%5Blabel%3D%22Nm%3D72%2C%20Ni%3D54%2C%20C%3D1.82432%22%5D%3B%20%22frame123.jpg%22%20--%20%22frame124.jpg%22%5Blabel%3D%22Nm%3D74%2C%20Ni%3D51%2C%20C%3D1.68874%22%5D%3B%20%22frame124.jpg%22%20--%20%22frame125.jpg%22%5Blabel%3D%22Nm%3D75%2C%20Ni%3D46%2C%20C%3D1.5082%22%5D%3B%20%22frame125.jpg%22%20--%20%22frame126.jpg%22%5Blabel%3D%22Nm%3D91%2C%20Ni%3D77%2C%20C%3D2.1813%22%5D%3B%20%22frame126.jpg%22%20--%20%22frame127.jpg%22%5Blabel%3D%22Nm%3D89%2C%20Ni%3D75%2C%20C%3D2.16138%22%5D%3B%20%22frame127.jpg%22%20--%20%22frame128.jpg%22%5Blabel%3D%22Nm%3D89%2C%20Ni%3D70%2C%20C%3D2.01729%22%5D%3B%20%22frame128.jpg%22%20--%20%22frame129.jpg%22%5Blabel%3D%22Nm%3D73%2C%20Ni%3D68%2C%20C%3D2.27425%22%5D%3B%20%22frame129.jpg%22%20--%20%22frame130.jpg%22%5Blabel%3D%22Nm%3D71%2C%20Ni%3D59%2C%20C%3D2.01365%22%5D%3B%20%22frame130.jpg%22%20--%20%22frame131.jpg%22%5Blabel%3D%22Nm%3D68%2C%20Ni%3D54%2C%20C%3D1.90141%22%5D%3B%20%22frame131.jpg%22%20--%20%22frame132.jpg%22%5Blabel%3D%22Nm%3D70%2C%20Ni%3D54%2C%20C%3D1.86207%22%5D%3B%20%22frame132.jpg%22%20--%20%22frame133.jpg%22%5Blabel%3D%22Nm%3D77%2C%20Ni%3D64%2C%20C%3D2.05788%22%5D%3B%20%22frame133.jpg%22%20--%20%22frame134.jpg%22%5Blabel%3D%22Nm%3D89%2C%20Ni%3D69%2C%20C%3D1.98847%22%5D%3B%20%22frame134.jpg%22%20--%20%22frame135.jpg%22%5Blabel%3D%22Nm%3D92%2C%20Ni%3D63%2C%20C%3D1.76966%22%5D%3B%20%22frame135.jpg%22%20--%20%22frame136.jpg%22%5Blabel%3D%22Nm%3D82%2C%20Ni%3D55%2C%20C%3D1.68712%22%5D%3B%20%22frame136.jpg%22%20--%20%22frame137.jpg%22%5Blabel%3D%22Nm%3D80%2C%20Ni%3D64%2C%20C%3D2%22%5D%3B%20%22frame137.jpg%22%20--%20%22frame138.jpg%22%5Blabel%3D%22Nm%3D99%2C%20Ni%3D81%2C%20C%3D2.14854%22%5D%3B%20%22frame138.jpg%22%20--%20%22frame139.jpg%22%5Blabel%3D%22Nm%3D96%2C%20Ni%3D81%2C%20C%3D2.20109%22%5D%3B%20%22frame139.jpg%22%20--%20%22frame140.jpg%22%5Blabel%3D%22Nm%3D99%2C%20Ni%3D65%2C%20C%3D1.72414%22%5D%3B%20%22frame140.jpg%22%20--%20%22frame141.jpg%22%5Blabel%3D%22Nm%3D104%2C%20Ni%3D79%2C%20C%3D2.01531%22%5D%3B%20%22frame141.jpg%22%20--%20%22frame142.jpg%22%5Blabel%3D%22Nm%3D97%2C%20Ni%3D76%2C%20C%3D2.04852%22%5D%3B%20%22frame142.jpg%22%20--%20%22frame143.jpg%22%5Blabel%3D%22Nm%3D92%2C%20Ni%3D66%2C%20C%3D1.85393%22%5D%3B%20%22frame143.jpg%22%20--%20%22frame144.jpg%22%5Blabel%3D%22Nm%3D92%2C%20Ni%3D74%2C%20C%3D2.07865%22%5D%3B%20%22frame144.jpg%22%20--%20%22frame145.jpg%22%5Blabel%3D%22Nm%3D100%2C%20Ni%3D76%2C%20C%3D2%22%5D%3B%20%22frame145.jpg%22%20--%20%22frame146.jpg%22%5Blabel%3D%22Nm%3D99%2C%20Ni%3D67%2C%20C%3D1.77719%22%5D%3B%20%22frame146.jpg%22%20--%20%22frame147.jpg%22%5Blabel%3D%22Nm%3D109%2C%20Ni%3D89%2C%20C%3D2.18673%22%5D%3B%20%22frame147.jpg%22%20--%20%22frame148.jpg%22%5Blabel%3D%22Nm%3D132%2C%20Ni%3D96%2C%20C%3D2.01681%22%5D%3B%20%22frame148.jpg%22%20--%20%22frame149.jpg%22%5Blabel%3D%22Nm%3D135%2C%20Ni%3D93%2C%20C%3D1.91753%22%5D%3B%20%22frame149.jpg%22%20--%20%22frame150.jpg%22%5Blabel%3D%22Nm%3D131%2C%20Ni%3D88%2C%20C%3D1.86047%22%5D%3B%20%22frame150.jpg%22%20--%20%22frame151.jpg%22%5Blabel%3D%22Nm%3D152%2C%20Ni%3D101%2C%20C%3D1.88433%22%5D%3B%20%22frame151.jpg%22%20--%20%22frame152.jpg%22%5Blabel%3D%22Nm%3D141%2C%20Ni%3D100%2C%20C%3D1.98807%22%5D%3B%20%22frame152.jpg%22%20--%20%22frame153.jpg%22%5Blabel%3D%22Nm%3D141%2C%20Ni%3D96%2C%20C%3D1.90855%22%5D%3B%20%22frame153.jpg%22%20--%20%22frame154.jpg%22%5Blabel%3D%22Nm%3D118%2C%20Ni%3D86%2C%20C%3D1.98157%22%5D%3B%20%22frame154.jpg%22%20--%20%22frame155.jpg%22%5Blabel%3D%22Nm%3D103%2C%20Ni%3D82%2C%20C%3D2.10797%22%5D%3B%20%22frame155.jpg%22%20--%20%22frame156.jpg%22%5Blabel%3D%22Nm%3D100%2C%20Ni%3D59%2C%20C%3D1.55263%22%5D%3B%20%22frame156.jpg%22%20--%20%22frame157.jpg%22%5Blabel%3D%22Nm%3D110%2C%20Ni%3D78%2C%20C%3D1.90244%22%5D%3B%20%22frame157.jpg%22%20--%20%22frame158.jpg%22%5Blabel%3D%22Nm%3D111%2C%20Ni%3D73%2C%20C%3D1.76755%22%5D%3B%20%22frame158.jpg%22%20--%20%22frame159.jpg%22%5Blabel%3D%22Nm%3D149%2C%20Ni%3D88%2C%20C%3D1.66983%22%5D%3B%20%22frame159.jpg%22%20--%20%22frame160.jpg%22%5Blabel%3D%22Nm%3D144%2C%20Ni%3D109%2C%20C%3D2.12891%22%5D%3B%20%22frame160.jpg%22%20--%20%22frame161.jpg%22%5Blabel%3D%22Nm%3D151%2C%20Ni%3D96%2C%20C%3D1.80113%22%5D%3B%20%22frame161.jpg%22%20--%20%22frame162.jpg%22%5Blabel%3D%22Nm%3D145%2C%20Ni%3D87%2C%20C%3D1.68932%22%5D%3B%20%22frame162.jpg%22%20--%20%22frame163.jpg%22%5Blabel%3D%22Nm%3D131%2C%20Ni%3D71%2C%20C%3D1.50106%22%5D%3B%20%22frame163.jpg%22%20--%20%22frame164.jpg%22%5Blabel%3D%22Nm%3D123%2C%20Ni%3D80%2C%20C%3D1.78174%22%5D%3B%20%22frame164.jpg%22%20--%20%22frame165.jpg%22%5Blabel%3D%22Nm%3D97%2C%20Ni%3D65%2C%20C%3D1.75202%22%5D%3B%20%22frame165.jpg%22%20--%20%22frame166.jpg%22%5Blabel%3D%22Nm%3D115%2C%20Ni%3D65%2C%20C%3D1.52941%22%5D%3B%20%22frame166.jpg%22%20--%20%22frame167.jpg%22%5Blabel%3D%22Nm%3D112%2C%20Ni%3D64%2C%20C%3D1.53846%22%5D%3B%20%22frame167.jpg%22%20--%20%22frame168.jpg%22%5Blabel%3D%22Nm%3D110%2C%20Ni%3D73%2C%20C%3D1.78049%22%5D%3B%20%22frame168.jpg%22%20--%20%22frame169.jpg%22%5Blabel%3D%22Nm%3D102%2C%20Ni%3D61%2C%20C%3D1.58031%22%5D%3B%20%22frame169.jpg%22%20--%20%22frame170.jpg%22%5Blabel%3D%22Nm%3D95%2C%20Ni%3D50%2C%20C%3D1.36986%22%5D%3B%20%22frame170.jpg%22%20--%20%22frame171.jpg%22%5Blabel%3D%22Nm%3D124%2C%20Ni%3D69%2C%20C%3D1.52655%22%5D%3B%20%22frame171.jpg%22%20--%20%22frame172.jpg%22%5Blabel%3D%22Nm%3D112%2C%20Ni%3D68%2C%20C%3D1.63462%22%5D%3B%20%22frame172.jpg%22%20--%20%22frame173.jpg%22%5Blabel%3D%22Nm%3D88%2C%20Ni%3D59%2C%20C%3D1.71512%22%5D%3B%20%22frame173.jpg%22%20--%20%22frame174.jpg%22%5Blabel%3D%22Nm%3D86%2C%20Ni%3D56%2C%20C%3D1.6568%22%5D%3B%20%22frame174.jpg%22%20--%20%22frame175.jpg%22%5Blabel%3D%22Nm%3D84%2C%20Ni%3D60%2C%20C%3D1.80723%22%5D%3B%20%22frame175.jpg%22%20--%20%22frame176.jpg%22%5Blabel%3D%22Nm%3D90%2C%20Ni%3D74%2C%20C%3D2.11429%22%5D%3B%20%22frame176.jpg%22%20--%20%22frame177.jpg%22%5Blabel%3D%22Nm%3D98%2C%20Ni%3D60%2C%20C%3D1.60428%22%5D%3B%20%7D%0A%20)
